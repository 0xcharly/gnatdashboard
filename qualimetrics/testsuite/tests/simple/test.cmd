# launch qualimetrics on a trivial project
qualimetrics -Pdefault

# verify that the object directory contains the expected files
# at the expected locations
find obj/ | sort
