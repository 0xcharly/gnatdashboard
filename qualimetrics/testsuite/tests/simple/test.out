obj/
obj/hello.adb.metrix
obj/metrix.xml
obj/qualimetrics
obj/qualimetrics/logs
obj/qualimetrics/logs/gnatmetric.log
obj/qualimetrics/qualimetrics.db
